### PR TITLE
Update spanish translation as requested in issue #322

### PR DESCRIPTION
**Pull request by huguei**
## Purpose

This PR updates spanish translation to keep in sync with develop branch.
Please note poedit tool reorder the messages so it appears more changes than effective.

## Context

#322 

## Changes

share/es.po

### CODE REVIEW

**Review by matsduf — COMMENTED** [review]
I see no issues.

**Comment by huguei** [comment]
I did a second check and it seems ready to go. Please proceed.

**Review by matsduf — APPROVED** [review]

File: share/es.po [hunk 1]
```diff
@@ -221,6 +264,19 @@ msgstr "Caso de prueba"
 msgid "Message"
 msgstr "Mensaje"
 
+msgid ""
+"Only one domain can be given for testing. Did you forget to prepend an "
+"option with '--<OPTION>'?\n"
+msgstr ""
+"Solo puede indicarse 1 dominio para las pruebas. ¿Quizás incluyó una opción "
```
[the remark this answers is not in the document]
Comment (matsduf) [reply share/es.po:271]: @huguei, please comment. Should it be updated? If yes, I can easily do it if you write to what.

[the remark this answers is not in the document]
Comment (huguei) [reply share/es.po:271]: Thanks for you careful review @tgreenx , but no, it's clearer with a number. In Spanish the word "un" can indicate an indefinite article ("a/an") or the number 1, and in this case saying "Solo puede indicarse un dominio para las pruebas" is pretty vague. Sounds equivalent to "only any domain can be given for testing". In contrast, "only 1 domain can be indicated for testing" is unambiguous that it refers to quantity!

[the remark this answers is not in the document]
Comment (matsduf) [reply share/es.po:271]: Thanks, @huguei! I will merge.

**Pull request merged**